File: tarea 3/ejercicio3.java
// recorrido de arreglo
public class recorrido {
    public static void main(string[] args) {
        int[] numeros = {10, 20, 30, 40};

        // for normal
        for (int i = 0; i < numeros.length; i++) {
            system.out.println(numeros[i]);
        }

        // for each
        for (int n : numeros) {
            system.out.println(n);
        }
    }
}

File: tarea 3/ejercicio3.cpp
// recorrido de arreglo
#include <iostream>
using namespace std;

int main() {
    int numeros[] = {10, 20, 30, 40};
    int tamaño = sizeof(numeros)/sizeof(numeros[0]);

    // for normal
    for (int i = 0; i < tamaño; i++) {
        cout << numeros[i] << endl;
    }

    // range-based for (c++11 en adelante)
    for (int n : numeros) {
        cout << n << endl;
    }

    return 0;
}